### PR TITLE
396 properly enqueue scripts and styles

### PR DESCRIPTION
**Pull request by boonedev**
This PR:
 - Removes enqueue-scripts.php and replaces it with ECAD\Admin\Enqueue_Admin and EDAC\Inc\Enqueue_Frontend
 - Adds maybe_enqueue_email_opt_in_script() to EDAC/Admin/Enqueue_Admin
 - Adds js & css for emailOptIn to webpack build 
 - Removes zoho form from Edac\Admin\Welcome_Page and replaces with it with ActiveCampaign

### CODE REVIEW

File: accessibility-checker.php [hunk 1]
```diff
@@ -134,9 +133,8 @@
 /**
  * Filters and Actions
  */
-add_action( 'admin_enqueue_scripts', 'edac_admin_enqueue_scripts' );
-add_action( 'admin_enqueue_scripts', 'edac_admin_enqueue_styles' );
-add_action( 'wp_enqueue_scripts', 'edac_enqueue_scripts' );
+add_action( 'admin_enqueue_scripts', array( 'EDAC\Admin\Enqueue_Admin', 'enqueue' ) );
```
Comment (SteveJonesDev) [inline accessibility-checker.php:136]: Shouldn't this be moved to the `class-admin.php` so it's never executed unless in the admin, consistency, and to keep the main plugin file clean?

File: accessibility-checker.php [hunk 2]
```diff
@@ -134,9 +133,8 @@
...
+add_action( 'wp_enqueue_scripts', array( 'EDAC\Inc\Enqueue_Frontend', 'enqueue' ) );
```
Comment (SteveJonesDev) [inline accessibility-checker.php:137]: Shouldn't this be moved to the `class-plugin.php` so it's never executed unless on the front-end, consistency, and to keep the main plugin file clean?

**Review by SteveJonesDev — CHANGES_REQUESTED** [review]
Just a couple of changes.

File: accessibility-checker.php (hunk 2 above)
Comment (boonedev) [reply accessibility-checker.php]: @SteveJonesDev Good catch, thanks.

**Review by SteveJonesDev — APPROVED** [review]

**Pull request merged**